WeartGloves/README.md

```diff
@@ -0,0 +1,38 @@
+# Instruction on how to install and use the Weart Glove Device
+
+## Build and Install
+
+1. Install the `WeArtGloveLib` following the instruction [here](https://github.com/ami-iit/WEART-SDK-Cpp/tree/devel_ami-iit#ami-iit).
+2. clone the repo `git clone https://github.com/ami-iit/yarp-device-haptic-gloves.git`
+3. `git checkout devel_WeartGlove`
+4. `mkdir && cd build`
+5. `cmake -G"Visual Studio 17 2022" ..`
+6. Notice that the files are going to be installed to `C:\Program Files (x86)\YOUR_PROJECT_NAME\bin`
+    - If you want to change `CMAKE_INSTALL_PREFIX`, run `ccmake ..`
+    - In our case: `C:<PATH_TO_ROBOTOLOGYSUPERBUILD>\build\install`
+7. `cmake --build . --target INSTALL --config Release`
+8. Download and Install the latest WEARTMiddleware [link](https://weart.it/developer-guide/).
```

Comment:
Can we report the latest tested version?

Reply:
Yes. Also need to to note that so far it is only possible to download the latest version from their website.

Reply:
If we can download a file, I would store it somewhere in our internal sharepoint for reproducibility.

Reply:
> If we can download a file, I would store it somewhere in our internal sharepoint for reproducibility.

I have stored all the files we received from WeArt Team here: [SharePoint](https://istitutoitalianotecnologia.sharepoint.com/:f:/r/sites/ArtificialandMechanicalIntelligence/Documenti%20condivisi/Team%20tau/WeArt%20Gloves?csf=1&web=1&e=MHI4If)